rotkehlchen/api/rest.py

```diff
@@ -407,7 +435,13 @@ def query_tasks_outcome(self, task_id: Optional[int]) -> Response:
         }
         return api_response(result=result_dict, status_code=HTTPStatus.NOT_FOUND)
 
-    def _get_exchange_rates(self, given_currencies: list[AssetWithOracles]) -> dict[str, Any]:
+    @asynchronous()
+    def get_exchange_rates(self, given_currencies: list[AssetWithOracles]) -> dict[str, Any]:
+        if len(given_currencies) == 0:
+            return wrap_in_fail_result(
+                message='Empty list of currencies provided',
+                status_code=HTTPStatus.BAD_REQUEST,
```

Comment:
Is this missing from the docs? Add it if needed

Reply:
Do you mean the api .rst docs file? If you mean that one, then this is mentioned there https://github.com/rotki/rotki/blob/e3a0c09c83eae4cb8ed28d676b7d60cc4107b3bd/docs/api.rst#L1347